app/assets/stylesheets/application.css.scss

```diff
@@ -12,4 +12,45 @@
  *
  *= require_tree .
  *= require_self
+ *= require foundation_and_overrides
```

Comment:
Hmmm ... this seems like a strange ordering.  Were you not able to require foundation, then self, then tree?